### DISCUSSION TITLE
How do I drag the thumb on mobile device?

### DISCUSSION THREAD

**Discussion by xclloe00** [idiotWu/smooth-scrollbar#543] [Q&A]
On desktop, dragging the thumb element works perfect but on mobile devices (iPhone > Chrome) dragging the thumb element does not work. How can I make it work?

Basically, when I try to drag it down it goes up and viceversa.

**Comment by sadeghbarati** [comment]
https://github.com/idiotWu/smooth-scrollbar/blob/66c67b85d35c14486bd25a73806c0ab13ffeb267/src/events/mouse.ts#L84C3-L87C6

Only `mousedown` event is registered on scrollbar tracks (thumb)
On mobile devices, it works only with `touch`
